angular/projects/admin-nrpti/src/app/records/inspections/inspection-detail/inspection-detail.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { takeUntil } from 'rxjs/operators';
+import { Inspection } from '../../../../../../common/src/app/models/master/inspection';
+import { Subject } from 'rxjs';
+import { ActivatedRoute, Router } from '@angular/router';
+import { RecordComponent } from '../../utils/record-component';
+import { RecordUtils } from '../../utils/record-utils';
+
+@Component({
+  selector: 'app-inspection-detail',
+  templateUrl: './inspection-detail.component.html',
+  styleUrls: ['./inspection-detail.component.scss']
+})
+export class InspectionDetailComponent extends RecordComponent implements OnInit, OnDestroy {
+  private ngUnsubscribe: Subject<boolean> = new Subject<boolean>();
+
+  public isLoading = true;
```

Comment:
This is one of many places where isLoading is not used in the UI at all. Should be tossed.  The only place that uses this paradigm is the map.

Reply:
Removed